include/buildpack.bash

```diff
@@ -151,28 +174,53 @@ buildpack-execute() {
 		title "Unable to select a buildpack"
 		exit 1
 	fi
+}
 
-	cd "$build_path"
-	unprivileged "$selected_path/bin/compile" "$build_path" "$cache_path" "$env_path"
-	if [[ -f "$selected_path/bin/release" ]]; then
-		unprivileged "$selected_path/bin/release" "$build_path" "$cache_path" > "$build_path/.release"
-	fi
-	if [[ -f "$build_path/.release" ]]; then
-		config_vars="$(cat $build_path/.release | yaml-get config_vars)"
-		if [[ "$config_vars" ]]; then
-			mkdir -p $build_path/.profile.d
-			OIFS=$IFS
-			IFS=$'\n'
-			for var in $config_vars; do
-				echo "export $(echo $var | sed -e 's/=/="/' -e 's/$/"/')" >> "$build_path/.profile.d/00_config_vars.sh"
-			done
-			IFS=$OIFS
-		fi
-	fi
-	cd - > /dev/null
-
+move-build-to-app() {
```

Comment:
let's move this up near `_envfile-parse()` and change the function name to `_move-build-to-app`. the pattern here is unexported functions begin with `_` and are grouped with other unexported functions.